### DISCUSSION TITLE
Error after microcks installation using gcp gke, the keycloak admin console is showing the loading page only.

### DISCUSSION THREAD

**Discussion by Athira31** [microcks/microcks#1014] [Q&A]
I tried installing microcks using gcp, I had created a kubernetes cluster in gke and installed microcks using the helm chart. When I tried to access the urls created, I'm getting " Failed to initialize authentication subsystem" and the keycloak url is not coming up showing "Loading the admin console". 
How can this issue be resolved?

**Comment by yada** [comment]
Hi,
If you have done the default helm installation, this is certainly because you have not validated the auto generated SSL certificates in your web browser. Please let us know if it is the case and if it helps ;)
Regards,
Yacine
